active-directory-new-domain/azuredeploy.json

```diff
@@ -253,10 +143,10 @@
             "name": "[variables('adRDPNAT')]",
             "properties": {
               "frontendIPConfiguration": {
-                "id": "[variables('adlbFEConfigID')]"
+                "id": "[concat(resourceId('Microsoft.Network/loadBalancers', variables('adLBName')), '/frontendIPConfigurations/', variables('adLBFE'))]"
```

Comment:
If this is really a resourceId can it be simplified?  e.g.

resourceId('Microsoft.Network/loadBalancers/frontendIPConfigurations', variables('adLBName'), variables('adLBFE'))

IOW remove concat